mesa_frames/concrete/mixin.py

```
@@ -43,16 +43,18 @@ def some_method(self):
-from typing import Literal
+from __future__ import annotations
 
+from collections.abc import Callable, Collection, Hashable, Iterator, Sequence
 import polars as pl
-from typing_extensions import Any, overload
+from beartype import beartype
+from typing import Any, overload, Literal
 
 from mesa_frames.abstract.mixin import DataFrameMixin
-from mesa_frames.types_ import DataFrame, PolarsMask
+from mesa_frames.types_ import DataFrame, PolarsMask, PolarsDataFrameInput, PolarsIndex
```

Ruff (0.8.2)

`mesa_frames.types_.DataFrame` imported but unused

Remove unused import: `mesa_frames.types_.DataFrame`

(F401)